util/compaction_job_stats_impl.cc

```diff
@@ -20,6 +20,7 @@ void CompactionJobStats::Reset() {
   num_output_records = 0;
   num_output_files = 0;
 
+  is_full_compaction = 0;
```

Comment:
Would be nicer to initialize these two using `false` (or else the linter might complain).